code/modules/surgery/organs/organ_external.dm

```diff
@@ -552,6 +523,52 @@ Note that amputating the affected organ does in fact remove the infection from t
 			qdel(src) // If you flashed away to ashes, YOU FLASHED AWAY TO ASHES
 			return null
 
+/obj/item/organ/external/chest/droplimb()
+	if(cannot_amputate || !owner)
+		return
+
+	var/mob/living/carbon/C = owner
+
+	var/organ_spilled = 0
+	var/turf/T = get_turf(C)
+	C.add_splatter_floor(T)
+	playsound(get_turf(C), 'sound/effects/splat.ogg', 25, 1)
+	for(var/X in C.internal_organs)
+		var/obj/item/organ/O = X
+		var/org_zone = check_zone(O.parent_organ)
+		if(org_zone != "chest")
+			continue
+		O.remove(C)
+		O.forceMove(T)
+		organ_spilled = 1
+
+	if(organ_spilled)
+		C.visible_message("<span class='danger'><B>[C]'s internal organs spill out onto the floor!</B></span>")
+	return 1
+
+
+/obj/item/organ/external/attackby(obj/item/W, mob/user, params)
```

Comment:
`obj/item/I` is usually used for this no?

Reply:
its kinda just a stylization thing but i can change it later.

Reply:
I just remember Fox bringing it up on a PR a while back is why I mention it.